### PR TITLE
fix cpu reservation code

### PR DESCRIPTION
**Pull request by sjenning**
Fixes segfault with `kletGetter.GetNode()` returning nil.  `GetNode()` doesn't work until the status manager is up and the node has reported status to the API server.

### CODE REVIEW

File: pkg/kubelet/cpumanager/cpu_manager.go [hunk 1]
```diff
@@ -100,9 +102,14 @@ func NewManager(cpuPolicyName string, cr runtimeService, kletGetter kletGetter,
 		}
 		glog.Infof("[cpumanager] detected CPU topology: %v", topo)
 
-		// Get node to figure out reserved CPUs
-		node, err := kletGetter.GetNode()
-		topo.NumReservedCores = getReservedCPUs(node.Status)
+		resources := capacityProvider.GetNodeAllocatableReservation()
+		glog.Errorf("[cpumanager] resources: %v", resources)
```
Comment (ConnorDoyle) [inline pkg/kubelet/cpumanager/cpu_manager.go]: `Infof` instead?

File: pkg/kubelet/cpumanager/cpu_manager.go [hunk 2]
```diff
@@ -100,9 +102,14 @@ func NewManager(cpuPolicyName string, cr runtimeService, kletGetter kletGetter,
...
+		cpuResource, ok := resources[v1.ResourceCPU]
+		if ok {
+			reservedCores := int(cpuResource.Value())
+			glog.Errorf("[cpumanager] reserving %v cores due to kube/system reserved", reservedCores)
```
Comment (ConnorDoyle) [inline pkg/kubelet/cpumanager/cpu_manager.go]: `Infof`?

**Review by ConnorDoyle — APPROVED** [review]
LGTM. Log level nits, either fix or drop then will merge.

File: pkg/kubelet/cpumanager/cpu_manager.go (hunk 1 above)
Comment (sjenning) [reply pkg/kubelet/cpumanager/cpu_manager.go]: this was debug.  just dropped it.

**Comment by sjenning** [comment]
Updated

**Review by ConnorDoyle — APPROVED** [review]
:100:

**Pull request merged**